<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PMainOptionsDlg</class>
 <widget class="QDialog" name="PMainOptionsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>302</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PoePal Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="PoePal.qrc">
    <normaloff>:/PoePal/Resources/logo.png</normaloff>:/PoePal/Resources/logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="_Tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="_GeneralTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="_ToolbarDisplayLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Toolbar Display Mode: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="_ToolbarDisplayCombo">
         <item>
          <property name="text">
           <string>Icon Only</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Text Beside Icon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Text Under Icon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Text Only</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="_GeneralSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_ChatTab">
      <attribute name="title">
       <string>Chat</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="PChatWidgetsWidget" name="_ChatWidgets" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_ShortcutsTab">
      <attribute name="title">
       <string>Shortcuts</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="_ShortcutsScroll">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="_ScrollAreaLayout">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>356</width>
            <height>229</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTreeWidget" name="_ShortcutsTree">
             <property name="midLineWidth">
              <number>2</number>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
             <property name="columnCount">
              <number>4</number>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string notr="true">2</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string notr="true">3</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string notr="true">4</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="_ButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PChatWidgetsWidget</class>
   <extends>QWidget</extends>
   <header>PChatWidgetsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PoePal.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>_ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>PMainOptionsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>PMainOptionsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
